react/components/Charts/LineChart/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+import {
+    Line,
+    LineChart as LineChartBase,
+    XAxis,
+    YAxis,
+    Tooltip,
+    CartesianGrid,
+    ResponsiveContainer,
+} from 'recharts'
+import { colors, defaultProps } from './constants'
+import useChart from '../hooks/useChart'
+
+interface Props {
+    data: any,
+    dataKeys: string[],
+    xAxisKey: string,
+    yAxisKey?: string,
+    schema: ChartProps
+}
+
+const renderLines = (keys: string[]) => {
+    return keys.map((key, i) => 
```

Comment:
You do not need this `return`

Reply:
`const renderLines = (keys: string[]) => (keys.map((key, i) => ....))`